roles/blog+catalog/templates/www.librivox.org.conf

```diff
@@ -0,0 +1,17 @@
+<VirtualHost *:443>
+                                                                                                
+    ServerName www.librivox.org
+    Redirect permanent / https://librivox.org/
+
+    SSLEngine On
+    SSLCertificateFile /etc/letsencrypt/live/librivox.org/fullchain.pem
+    SSLCertificateKeyFile /etc/letsencrypt/live/librivox.org/privkey.pem
+
+</VirtualHost>
+
+<VirtualHost *:80>
+
+    ServerName www.librivox.org
```

Comment:
Not 100% sure, but I suspect the same can be achieved by just adding `ServerAlias www.librivox.org` to the existing config file for librivox.org. Are you able to try it in your localdev env? We still need the other stuff though.